packages/core/src/components/forms/_form-group.scss

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+@import "../../common/variables";
+@import "../../common/mixins";
+
+/*
+Form groups
+
+Form groups allow for more complex forms and fields than [simple labels](#components.forms.labels.simple-labels), for example,
+with [numeric inputs](#components.forms.numeric-input) or [control groups](#components.forms.control-group).
+They also allow the addition of helper text to aid with user navigation.
+
+<div class="pt-callout pt-intent-primary pt-icon-info-sign">
+  Note that this feature necessitates additional DOM elements, thus requiring a more complex structure achieved with `.pt-form-group`.
```

Comment:
this callout is confusing... "additional DOM elements" compared to what? i think it might be safe to remove.